var/spack/repos/builtin/packages/thrift/package.py

```diff
@@ -12,33 +12,45 @@ class Thrift(Package):
 
     version('0.9.2', '89f63cc4d0100912f4a1f8a9dee63678')
 
-    extends("python")
-
-    depends_on("autoconf")
-    depends_on("automake")
-    depends_on("bison")
-    depends_on("boost")
-    depends_on("flex")
-    depends_on("jdk")
-    depends_on("libtool")
-    depends_on("openssl")
-    depends_on("python")
-
-    # Compilation fails for most languages, fortunately cpp installs fine
-    # All other languages (yes, including C) are omitted until someone needs them
+    # Currently only support for c-family and python
+    variant('c', default=True, description="Build support for C-family languages")
+    variant('python', default=True, description="Build support for python")
+
+    depends_on('jdk')
+    depends_on('autoconf')
+    depends_on('automake')
+    depends_on('libtool')
+    depends_on('boost@1.53:')
+    depends_on('bison')
+    depends_on('flex')
+    depends_on('openssl')
+
+    # Variant dependencies
+    extends('python', when='+python')
+    depends_on('python', when='+python')
```

Comment:
You can get rid of this line. According to the [Spack documentation](http://software.llnl.gov/spack/packaging_guide.html#extensions):

> You can call either `depends_on` or `extends` on any one package, but not both. For example you cannot both `depends_on('python')` and `extends(python)` in the same package. `extends` implies `depends_on`.